src/util/cacheManager.ts

```diff
@@ -0,0 +1,66 @@
+import { Message } from 'discord.js';
+import { journalCache, gratefulnessCache} from '../data/channelCaches'
+
+/**
+ * @param msg The message object.
+ * @param channel The channel whose cache needs to be updated. 'j' stands for Journal and 'g' for Gratefulness
+ */
+export function addEntry(msg : Message, channel : 'j'|'g') : void{
+    const cache = resolveCache(channel)
+    const userID = msg.author.id
+    const currTime = new Date() //NOAH: Is this supposed to a Date Object or something like moment()?
```

Comment:
Yep, we'll need to localize the date according to the user that gets passed in. For now, just figure out how to get the `user` object into this function (see above comments).